tests/ci/cdk/cdk/codebuild/bm_framework_omnibus.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 0.2
+
+# Doc for batch https://docs.aws.amazon.com/codebuild/latest/userguide/batch-build-buildspec.html#build-spec.batch.build-list
+batch:
+  build-list:
+
+    - identifier: ubuntu2004_bm_framework
+      buildspec: ./tests/ci/codebuild/linux-x86/run_bm_framework.yml
+      env:
+        type: LINUX_CONTAINER
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_LARGE
+        image: ECR_REPO_PLACEHOLDER:ubuntu-20.04_clang-7x-bm-framework_latest
```

Comment:
Which platform? `ECR_REPO_PLACEHOLDER`

Reply:
ECR_REPO_PLACEHOLDER is used as a placeholder on lines 68-69 in `bm_framework_stack.py`. But in this case the platform will be the `aws-lc-ecr-linux-x86` repo.

Reply:
Discussed in slack. Standard CodeBuild container image can be used here. That means the custom repo prefix `ECR_REPO_PLACEHOLDER` is not needed.

Change is not required in this PR.